iphone/CoreApi/CoreApi/PlacePageData/Common/PlacePageInfoData.mm

```diff
@@ -79,6 +79,11 @@ - (instancetype)initWithRawData:(Info const &)rawData ohLocalization:(id<IOpenin
         case MetadataID::FMD_LEVEL: _level = ToNSString(value); break;
         case MetadataID::FMD_CAPACITY: _capacity = [NSString stringWithFormat:NSLocalizedString(@"capacity", nil), ToNSString(value)]; break;
         case MetadataID::FMD_WHEELCHAIR: _wheelchair = ToNSString(platform::GetLocalizedTypeName(value)); break;
+        case MetadataID::FMD_DRIVE_THROUGH:
+          if (value == "yes")
```

Comment:
Does metadata store a "yes" string? What are all possible values there? Storing a string is not as efficient as a byte/enum value, especially if the list of all valid values is limited.

Reply:
Yes, metadata is storing a "yes" string. Does any other metadata property store a byte/enum value? so I can check how it's coded?